src/Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -395,6 +395,17 @@ public void Stop()
                         return;
                     }
 
+                    if (blockValidationContext.Error == ConsensusErrors.BadTransactionDuplicate)
+                    {
+                        this.logger.LogTrace("(-)[BAD_TX_DUP]");
```

Comment:
There is already logging inside the rule no?

Reply:
Yes, true, I have now removed that